include/deal.II/grid/tria.h

```diff
@@ -939,7 +939,6 @@ namespace internal
  *       std::cout << "Triangulation has been refined." << std::endl;
  *     }
  *
- *     void f() {
```

Comment:
Now the layout of documentation is 
```
void f() {
  std::cout << "Triangulation has been refined." << std::endl;
}
void f() {
  Triangulation<dim> triangulation;
  // fill it somehow
  triangulation.signals.post_refinement.connect (&f);
  triangulation.refine_global (2);
```
I believe the second `void f() {` is undesired.